frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -233,7 +233,7 @@ export function DashCardVisualization({
   return (
     <WrappedVisualization
       className={cx(CS.flexFull, CS.overflowHidden, {
-        "pointer-events-none": isEditingDashboardLayout,
+        [CS.pointerEventsNone]: isEditingDashboardLayout,
```

Comment:
Make sure that you don't click the visualizations when editing the dashboard.
![Screenshot 2024-04-10 at 6 11 21 PM](https://github.com/metabase/metabase/assets/1937582/c8f716ec-7d07-40d7-a163-7ca830fd8e53)